src/plugins/auto/plugin.cpp

```diff
@@ -495,6 +495,9 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     std::mutex load_mutex;
     std::vector<Task> loads;
     std::once_flag readNetworkFlag;
+    std::string devCPUName;
+    int nCPUCores = parallel_get_max_threads();
```

Comment:
Check the nCPUCores on TGL/ADL/NUMA(XEON) if they are as expected (number of HW Cores)

Reply:
parallel_get_max_threads() gets 8 CPU Cores from TGL(machine: RIL DUT050-DG1MRB),  gets 4 CPU Cores from ADL(machine: RIL DUT4361-ADLN),  gets 64 CPU Cores from Xeon(R) Gold 6346(machine: odt-xiping-sonomacreek-01). The result matches CPU(s) which shows in lscpu.

Reply:
https://ark.intel.com/content/www/us/en/ark/products/212457/intel-xeon-gold-6346-processor-36m-cache-3-10-ghz.html

Reply:
getNumberOfCPUCores() 
gets 4 CPU Cores from TGL(machine: RIL DUT050-DG1MRB CPU:11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz, Socket(s):1, Core(s) per socket:4)
gets 6 CPU Cores from Alder Lake S (machine: RIL DUT103-ADLS  CPU:12th Gen Intel(R) Core(TM) i5-12400, Socket(s):1, Core(s) per socket:6), 
gets 32 CPU Cores from Xeon(R) Gold 6346(machine: odt-xiping-sonomacreek-01 CPU:Intel(R) Xeon(R) Gold 6346 CPU @ 3.10GHz, Socket(s):2, Core(s) per socket:16). 
The result matches CPU(s) which shows in cmd lscpu.